Source/WebInspectorUI/UserInterface/Views/SearchSidebarPanel.css

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2013-2020 Apple Inc. All rights reserved.
+ * Copyright (C) 2024 Frances Cornwall.
```

Comment:
Similar to other files, where individuals have contributed, can you add your email address to make it specific to you (i.e., `Frances Cornwall <abc@xyz.com>`). See below reference.

https://searchfox.org/wubkat/rev/4cfac5574bee015ca8149440b6658ffdb770ff0b/Source/WebCore/css/CSSSelector.h#2